### DISCUSSION TITLE
Details panel in result list

### DISCUSSION THREAD

**Discussion by Wookieech** [microsoft-search/pnp-modern-search#2973] [Q&A]
Is it possible to display the details-panel (three dots) in the result list?
![DetailsPane](https://github.com/microsoft-search/pnp-modern-search/assets/133745922/bde36a5e-169f-414e-a56a-66c3cecc7c87)

**Comment by wobba** [comment]
No easy way to do this. You would have to create a web component replicating the SharePoint UX.
